### DISCUSSION TITLE
$git </  bash: /home/../openbmc/scripts/git: No such file or directory || $apt-get install git</  git is already the newest version (1:2.34.1-1ubuntu1.10).

### DISCUSSION THREAD

**Discussion by wangliuusa** [community/community#79293] [Programming Help] [answered]
### Select Topic Area

Question

### Body

Sir / Madam,

Could you please help me in reinstall the deleted git ?

Sincerely,
Liu Wang
------------------------------------
Scenario:

rm -f -R openbmc-v08.07/
...
liuw@liuw-SYS-E100-12T-H:~/Downloads/OPENBMC-V08.07$ sudo apt-get install git
[sudo] password for liuw: 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
git is already the newest version (1:2.34.1-1ubuntu1.10).
0 upgraded, 0 newly installed, 0 to remove and 5 not upgraded.
liuw@liuw-SYS-E100-12T-H:~/Downloads/OPENBMC-V08.07$ 
...
liuw@liuw-SYS-E100-12T-H:~/Downloads/OPENBMC-V08.07$ git
bash: /home/liuw/Downloads/openbmc-v08.07/openbmc/scripts/git: No such file or directory

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @wangliuusa ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions.

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by faridnec** [comment]
first u can make sure it is already removed by
```bash
sudo apt-get purge git
```
This will remove Git and its configuration files. After that, reinstall Git:
```bash
sudo apt-get install git
```
navigate to your home directory and see if there are any remnants of the deleted directory:
```bash
cd ~
ls
```
Check if there's any remaining `openbmc-v08.07` directory. If it's still there, you can delete it using:
```bash
rm -rf openbmc-v08.07
```
try using git again:
```bash
git
```

**Comment by wangliuusa** [comment]
Hi Muhammad,

It is git-ing now !
Thank you for the detail.

Happy holiday.

Sincerely,
Liu Wang

**Reply by faridnec** [reply, marked as answer]
if it works, kindly mark the answer as an answer for the community if they have a similar problem  :) thanks. Happy holiday!

**Reply by wangliuusa** [reply]
Thank you.
